Jenkinsfile.labview

```diff
@@ -36,7 +36,7 @@ pipeline {
                 wrap([$class: 'Xvnc']){
                     withEnv(["HOME=${env.WORKSPACE}"]) {
                         sh """
-                            xdotool key --repeat 5 --delay 1000 Return | timeout 60 labview64 tests/testAllWithXmlReport.vi || true
+                            xdotool key --repeat 5 --delay 1000 Return | timeout 60 labview64 tests/testAllWithXmlReport.vi || true                            		    
```

Comment:
Please remove all white trails (spaces + tabs) after the `|| true`:
![image](https://user-images.githubusercontent.com/11474902/132734689-7158bd27-6b29-4a57-b395-45ac1e5cbe64.png)

Reply:
Done.